tools/run_tests/run_interop_tests.py

```diff
@@ -1029,5 +1096,8 @@ def aggregate_http2_results(stdout):
   dockerjob.finish_jobs([j for j in server_jobs.itervalues()])
 
   for image in docker_images.itervalues():
-    print('Removing docker image %s' % image)
-    dockerjob.remove_image(image)
+    if not args.manual_run:
+      print('Removing docker image %s' % image)
+      dockerjob.remove_image(image)
+    else:
+      print('Preserving docker image: %s' % image)
```

Comment:
sgtm.  If you could get the code in, I can also take it from here.  I have another PR that counts on the output of your code to move forward.  Thx.